content/docs/use-cases/versioning-data-and-models/index.md

```diff
@@ -0,0 +1,156 @@
+# Versioning Data and Models
+
+Taking data science from conceptual to applied requires answering data
+management questions around dataset and machine learning model versioning. How
+to track the evolution of data and ML models? How to organize and store the
+revisions of these data artifacts for easy access and sharing?
+
+It turns out that we can find some answers in software engineering, like
+[version control](https://en.wikipedia.org/wiki/Version_control). This means
+keeping a change history (commits), automatic file deduplication, enabling
+parallel teamwork (branching), peer-reviews (pull requests), etc. Imagine if we
+could do the same with data!
+
+![](/img/data-as-code.png) _Matching data descriptions to a working environment_
```

Comment:
See proposed figure in https://dvc-landing-use-cases-v-6d9wm9.herokuapp.com/doc/use-cases/versioning-data-and-models:

![image](https://user-images.githubusercontent.com/1477535/96793227-8ff8df80-13c1-11eb-9b59-c76dc9fe6601.png)